wave_kinematics/src/gimbal.cpp

```diff
@@ -0,0 +1,223 @@
+#include "wave/kinematics/gimbal.hpp"
+
+namespace wave {
+
+// Attitude Controller
+AttitudeController::AttitudeController(void) {
+    this->roll_controller = PID(0.3, 0.0, 0.2);
```

Comment:
Added doc to specify the Gimbal controller is assumed to run at 100hz, the default PID tunes are included as well.